x/distribution/keeper/msg_server.go

```diff
@@ -189,7 +189,7 @@ func (k msgServer) DepositValidatorRewardsPool(ctx context.Context, msg *types.M
 	}
 
 	if validator == nil {
-		return nil, errors.Wrap(types.ErrNoValidatorExists, msg.ValidatorAddress)
+		return nil, errors.Wrapf(types.ErrNoValidatorExists, "validator address: %s", msg.ValidatorAddress)
```

Comment:
Ditto (it looks like it was added fox fixing the linter, but it is better to just use wrap and don't repeat)